src/test/loader.ts

```diff
@@ -77,6 +77,12 @@ export class Loader {
   private _processConfigObject(rootDir: string) {
     validateConfig(this._config);
 
+    // Resolve script hooks relative to the root dir.
+    if (this._config.globalSetup)
+      this._config.globalSetup = path.resolve(rootDir, this._config.globalSetup);
```

Comment:
I believe this should be resolved to the config file dir not to the root dir.

Reply:
Root dir is the file dir (in this method :))